great_expectations/execution_engine/pandas_execution_engine.py

```diff
@@ -71,37 +72,35 @@
 
 
 class PandasExecutionEngine(ExecutionEngine):
```

Comment:
Just need the decorator here to get this to render
```suggestion
@public_api
class PandasExecutionEngine(ExecutionEngine):
```